src/test/java/core/basesyntax/db/StorageTest.java

```diff
@@ -0,0 +1,86 @@
+package core.basesyntax.db;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class StorageTest {
+    private final RegistrationService registrationService = new RegistrationServiceImpl();
+    private User firstUser;
+    private User secondUser;
+    private List<User> people = new ArrayList<>();
+
+    @BeforeEach
+    void setUp() {
+        firstUser = new User();
+        firstUser.setLogin("testLogin");
+        secondUser = new User();
+        secondUser.setLogin("testLogin");
+    }
+
+    @Test
+    void userWithSuchLogin() {
+        User user1 = new User();
+        user1.setLogin("User1234");
+
+        boolean userExists = false;
+        for (User user : people) {
```

Comment:
use StorageDao to find user

Reply:
But when I delete the last line, the mvn clean package throws the following error:
[ERROR] /Users/greqit/IdeaProjects/jv-registration-validation/src/test/java/core/basesyntax/StorageTest.java:1: File does not end with a newline. [NewlineAtEndOfFile]